addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -242,13 +242,14 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
 };
 
 HTMLLIElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false) {
-    if (offset > 0 || this.previousElementSibling) {
-        // If backspace inside li content or if the li is not the first one,
-        // it behaves just like in a normal element.
-        HTMLElement.prototype.oDeleteBackward.call(this, offset, alreadyMoved);
+    // If the deleteBackward is performed at the begening of a LI element,
+    // we take the current LI out of the list.
+    if (offset === 0) {
+        this.oToggleList(offset);
```

Comment:
Reading the commit message, I think this commit is going to clash functionally with this task: https://www.odoo.com/web#id=3442435&menu_id=4720&cids=1&action=333&active_id=1695&model=project.task&view_type=form

We need to discuss this together at GR with @SimonGenin and @hmai-odoo.

Reply:
as discuss on discord : 
I'll keep my PR as is and @hmai-odoo will adapt in his PR to take both behaviour into account.